rpcclient/README.md

```diff
@@ -23,15 +23,11 @@ implement and the API is not stable yet.
   Connects to a dcrd RPC server using TLS-secured websockets, registers for
   block connected and block disconnected notifications, and gets the current
   block count
-* [dcrwallet Websockets Example](https://github.com/decred/dcrd/tree/master/rpcclient/examples/dcrwalletwebsockets)  
```

Comment:
This can probably just be updated to point to the example in the dcrwallet repo since most people will probably start here and having a link to how to interact with wallet seems useful.